### ISSUE TITLE
Request for help for smoke test (Linux, Windows)

### ISSUE THREAD

**Issue by sltsheryl** [nus-cs2103-AY2223S1/forum#218]
Hi sorry for the trouble! Could anyone help to smoke test for Linux and Windows? I have packaged the app with the fonts, but I'm checking if it shows. The jar file is attached [here](https://github.com/sltsheryl/ip/releases/tag/A-Release). Thank you!

**Comment by RezwanArefin01** [comment]
Works as expected on Linux (Arch)! Fonts, images loaded. Resize also working properly. :+1: 
![Screenshot_20220916_163338](https://user-images.githubusercontent.com/22095441/190594446-bbfef9b0-38b5-4587-962e-a08fd64e3564.png)

**Comment by sltsheryl** [comment]
> Works as expected on Linux (Arch)! Fonts, images loaded. Resize also working properly. 👍 ![Screenshot_20220916_163338](https://user-images.githubusercontent.com/22095441/190594446-bbfef9b0-38b5-4587-962e-a08fd64e3564.png)

Thank you for the swift response!

**Comment by RezwanArefin01** [comment]
@sltsheryl By the way, can you give me a summery of how did you include the fonts? I would also like to include my fonts. I have some emojis that do not show up in other's PC. :disappointed:

**Comment by sltsheryl** [comment]
> @sltsheryl By the way, can you give me a summery of how did you include the fonts? I would also like to include my fonts. I have some emojis that do not show up in other's PC. 😞

Yep sure, I followed this [repo](https://github.com/TheItachiUchiha/FontLoad) to set custom fonts in JavaFX.  
1. I downloaded the font ttf file, add it to a `fonts` folder in `resources` folder.
2. I created a `Font.css` in a `css` folder under `resources`.
```
@font-face {
    src: url('../view/fonts/Monofur.ttf');
}
```
3. I used inline css to add the fonts in the respective DialogBox.fxml files in this way
`style="-fx-font-family: 'monofur for Powerline'; `in the `Label` tag
